cms/static/js/views/settings/main.js

```diff
@@ -1,6 +1,6 @@
 define(["js/views/validation", "codemirror", "underscore", "jquery", "jquery.ui", "js/utils/date_utils", "js/models/uploads",
-    "js/views/uploads", "js/utils/change_on_enter", "jquery.timepicker", "date"],
-    function(ValidatingView, CodeMirror, _, $, ui, DateUtils, FileUploadModel, FileUploadDialog, TriggerChangeEventOnEnter) {
+    "js/views/uploads", "js/utils/change_on_enter", "coffee/src/keyword_validator", "jquery.timepicker", "date"],
```

Comment:
`jquery.timepicker` and `data` don't appear to be used; they aren't passed as params to the function on the following line.

But maybe this is outside the scope and should be a separate PR to upstream?